guides/source/error_reporting.md

```diff
@@ -117,11 +126,32 @@ rescue StandardError => e
 end
 ```
 
-Any options you pass will be passed on the error subscribers.
+Any options you pass will be passed on to the error subscribers.
+
+#### Reporting Unexpected Errors
+
+You can report any errors that may unexpectedly occur when the imagined conditions of your code are not met by calling [`Rails.error.unexpected`](https://edgeapi.rubyonrails.org/classes/ActiveSupport/ErrorReporter.html#method-i-unexpected").
```

Comment:
"imagined conditions of your code" sounds a little odd. Maybe something like "You can report any unexpected error by calling..."